src/common/ui/style/index.js

```diff
@@ -2,15 +2,25 @@ import options from '../../options';
 import './style.css';
 
 let style;
+const CACHE_KEY = 'cacheCustomCSS';
+
+const setStyle = (css) => {
+  if (css && !style) {
+    style = document.createElement('style');
+    document.head.appendChild(style);
```

Comment:
To make this usable in **dev build** you need to put the style element after `<head>`:
```js
document.documentElement.appendChild(style);
```
because we're running early now and other components will add their styles after customCSS thus unpredictably messing up selectors with the same CSS specificity.

Reply:
I will change it to `document.documentElement.appendChild(style);`.